frontend/src/screens/MyTeam.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../App.global.scss';
+import { Text } from 'wix-style-react';
+
+const MyTeam = props => {
+  return (
+    <div className="MyTeam">
```

Comment:
This `className` seems more like an `id`. But in any case, why do we have this if we're not using it at the moment?